FAQ.md

```diff
@@ -0,0 +1,91 @@
+# FAQ for BBP spack usage
+
+
+<details>
+  <summary>Q: How do I build and *load* a piece of software?</summary>
+
+  We'll install `Ed(1)`, the standard editor.
+
+  Make sure you're setup as per [this](https://github.com/BlueBrain/spack#building-software-on-bluebrain5).
+
+  Specifically that you have the `spack` git repo on the `develop` branch,
+  and have created and linked the files into `~/.spack/`
+
+  ```console
+  spack install ed
+  ```
+
+  This produces output, and should end with something like:
+  `$SPACK_INSTALL_PREFIX/linux-rhel7-x86_64/gcc-6.4.0/ed-1.4-35jlkv`
+
+  One can then run `ed` with
+  ```
+  $SPACK_INSTALL_PREFIX/linux-rhel7-x86_64/gcc-6.4.0/ed-1.4-35jlkv/ed
+  ```
+
+  More complex packages will have an environment that needs to be setup by
+  the module system.
+  To find the module that was built, issue:
+  ```
+  spack module tcl find --full-path ed
+  ```
+
+  At which point, you should be able to:
```

Comment:
In order to get my modules accessible, I needed to:
```
MODULEPATH=/gpfs/bbp.cscs.ch/home/bcoste/software/modules/tcl/linux-rhel7-x86_64:${MODULEPATH}
```
otherwise I would get something like:
```
ModuleCmd_Load.c(213):ERROR:105: Unable to locate a modulefile for 'psp-validation/my-custom-version/python3/serial'
```

Reply:
Not necessarily :)
I ended up removing my ~/.spack/modules.yaml(created from the README instructions), only then I saw the module and managed to load it.

Reply:
Hm, @wizmer are modules inaccessible even when sourcing the Spack env?